drivers/sensor/realtek/rts5912/Kconfig

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2025, Realtek, SIBG-SD7
+# SPDX-License-Identifier: Apache-2.0
+
+config TACH_RTS5912
+	bool "RTS5912 Tachometer sensor"
+	default y
+	depends on DT_HAS_REALTEK_RTS5912_TACH_ENABLED
+	help
+	  Enable the Realtek RTS5912 tachometer sensor.
+
+if TACH_RTS5912
+
+choice
+	prompt "Number of tach edges"
+	default TACH_RTS5912_5_TACH_EDGES
+	help
+	  This value represents the number of Tach edges that
+	  will be used to determine the interval for which the number of
+	  100KHz pulses will be counted.
+
+config TACH_RTS5912_9_TACH_EDGES
+	bool "Configure 9 tach edges or 4 tach periods"
+
+config TACH_RTS5912_5_TACH_EDGES
+	bool "Configure 5 tach edges or 2 tach periods"
+
+config TACH_RTS5912_3_TACH_EDGES
+	bool "Configure 3 tach edges or 1 tach period"
+
+config TACH_RTS5912_2_TACH_EDGES
+	bool "Configure 2 tach edges or 1/2 tach period"
+
+endchoice
```

Comment:
This seems like it belongs with the dts bindings. Can you have 2 of these drivers with different number of edges? If so, you should put it in devicetree, kconfigs are global.